packages/components/psammead-section-label/src/index.jsx

```diff
@@ -4,63 +4,52 @@ import { bool, oneOf, shape, string } from 'prop-types';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 import {
   GEL_GROUP_2_SCREEN_WIDTH_MAX,
-  GEL_GROUP_3_SCREEN_WIDTH_MIN,
   MEDIA_QUERY_TYPOGRAPHY,
 } from '@bbc/gel-foundations/breakpoints';
 import {
-  GEL_SPACING,
-  GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
   GEL_SPACING_QUAD,
 } from '@bbc/gel-foundations/spacings';
-import { getDoublePica } from '@bbc/gel-foundations/typography';
-import { C_EBON, C_PEBBLE, C_WHITE } from '@bbc/psammead-styles/colours';
-import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { C_PEBBLE } from '@bbc/psammead-styles/colours';
 
-const halfLineHeightRem = group => group.lineHeight / 2 / 16;
+import { PlainTitle, LinkTitle } from './titles';
 
-const top = script => `
-  // place at middle of text line height
-  top: ${halfLineHeightRem(script.doublePica.groupA)}rem;
+const labelWrapperTopMargin = ({ script }) =>
+  script
+    ? `
+        margin-top: ${GEL_SPACING_QUAD};
 
-  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
-    top: ${halfLineHeightRem(script.doublePica.groupB)}rem;
+        ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+          margin-top: ${GEL_SPACING_TRPL};
+        }
+      `
+    : `margin-top: ${GEL_SPACING_QUAD};`;
+
+const Bar = styled.div`
+  border-top: 0.0625rem solid ${C_PEBBLE};
+  left: 0;
```

Comment:
Do we need `left` and `right` by default? Shouldn't they be inside the `   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} `?

Reply:
nice catch 👍 moved